<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiWidget</class>
 <widget class="QWidget" name="MultiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MultiWidget {
background: rgba(200, 200, 200);
}

#lowerFrame, #midFrame, #topFrame { 
padding: 3px;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0.0 rgba(95, 95, 95),
stop: 0.15 rgba(75, 75, 75),
stop: 0.9 rgba(95, 95, 95),
stop: 1.0 rgba(45, 45, 45));
border-top-left-radius: 4px;
border-top-right-radius: 4px;
border-bottom-left-radius: 4px;
border-bottom-right-radius: 4px;
}

QPushButton {
padding: 4px;
border-radius: 4px;
border: 1px solid rgba(55, 55, 55);
font-weight: bold;
/*
background: qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(25, 25, 25),
stop: 0.05 rgba(75, 75, 75),
stop: 0.15 rgba(125, 125, 125),
stop: 0.85 rgba(75, 75, 75),
stop: 1.0 rgba(25, 25, 25));
*/
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0.01 rgba(120, 120, 120),
stop: 0.05 rgba(150, 150, 150),
stop: 0.5 rgba(120, 120, 120),
stop: 0.52 rgba(80, 80, 80),
stop: 0.9 rgba(50, 50, 50),
stop: 1.0 rgba(50, 50, 50));
color: rgba(225, 225, 225);
}

QPushButton:pressed {
background: qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(25, 25, 25),
stop: 0.05 rgba(25, 25, 25),
stop: 0.15 rgba(75, 75, 75),
stop: 0.85 rgba(25, 25, 25),
stop: 1.0 rgba(0, 0, 0));
color: rgba(200, 200, 200);
}

QPushButton:disabled {
background:qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(50, 50, 50),
stop: 0.05 rgba(100, 100, 100),
stop: 0.15 rgba(150, 150, 150),
stop: 0.85 rgba(100, 100, 100),
stop: 1.0 rgba(50, 50, 50));
color: rgba(100, 100, 100);
}

QCheckBox {
background: transparent;
border: none;
}
/**
QPushButton {
border-radius: 4px;
border: 1px solid rgba(75, 75, 75);
background: qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(25, 25, 25),
stop: 0.05 rgba(75, 75, 75),
stop: 0.15 rgba(125, 125, 125),
stop: 0.85 rgba(75, 75, 75),
stop: 1.0 rgba(25, 25, 25));
color: rgba(225, 225, 225);
}
/**
QPushButton:pressed {
background:qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(0, 0, 0),
stop: 0.05 rgba(50, 50, 50),
stop: 0.15 rgba(100, 100, 100),
stop: 0.85 rgba(50, 50, 50),
stop: 1.0 rgba(0, 0, 0));
color: rgba(200, 200, 200);
}**/

#controlsFrame {
	background: rgba(200, 200, 200);
border-top-left-radius: 4px;
border-top-right-radius: 4px;
border-bottom-left-radius: 2px;
border-bottom-right-radius: 2px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
stop: 0.0 rgba(125, 125, 125),
stop: 0.02 rgba(200, 200, 200),
stop: 0.98 rgba(200, 200, 200),
stop: 1.0 rgba(125, 125, 125));
}

QLineEdit {
border: 1px solid rgba(175, 175, 175);
background: rgba(255, 255, 255);
border-top-left-radius: 4px;
border-top-right-radius: 4px;
border-bottom-right-radius: 4px;
border-bottom-left-radius: 4px;
}

QGroupBox {
border: 1px solid rgba(175, 175, 175);
border-radius: 4px;
margin-top: 1ex;
font-weight: bold;
color: white;
}

QGridLayout {
border: 1px solid rgba(175, 175, 175);
border-radius: 4px;
margin: 10px;
}

#simulatorButton {
width: 100px;
}

QLabel {
color: white;
font-weight: bold;
}

#windFrame{
width: 200px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="topFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QWebView" name="gemap">
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="midFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QTextEdit" name="logEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="windFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QGroupBox" name="connectionGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>80</height>
            </size>
           </property>
           <property name="title">
            <string>Connection</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="portLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="portEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximum">
               <number>20000</number>
              </property>
              <property name="value">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="startServerButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Start server</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="lowerFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="controlsBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>false</underline>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="title">
         <string>Simulation controls</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="startSimulationButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Start simulation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="selectWinnerButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Select winner</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="resetSimulationButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Reset simulation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>--&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>--&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QPushButton" name="XMLFileButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Select XML File</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="spinnakerBox">
           <property name="text">
            <string>Spinnaker</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>startServer()</slot>
 </slots>
</ui>
